moving_nest/fv_moving_nest_main.F90

```diff
@@ -18,1150 +22,1219 @@
 !* or see:   http://www.gnu.org/licenses/gpl.html                      *
 !***********************************************************************
 
-!***********************************************************************
-!> @file
-!! @brief Provides top-level interface for moving nest functionality
-!! @author W. Ramstrom, AOML/HRD   05/27/2021
-!! @email William.Ramstrom@noaa.gov
-! =======================================================================!
-
+!> @brief Provides top-level interface for moving nest functionality.
+!>
+!> @author W. Ramstrom, AOML/HRD (William.Ramstrom@noaa.gov) @date  05/27/2021
 module fv_moving_nest_main_mod
 
-#include <fms_platform.h>
```

Comment:
Why are there so many code changes in this file?

Reply:
I'm trying to investigate why it's showing up this way. I double checked this file in your repo and compared it to develop. It showed zero code changes, but after swapping the files for some reason it's showing up as huge chunks of changes. I followed the same procedure as the other 2 files.

Reply:
Much better, thanks!